asv_bench/benchmarks/benchmarks_hindcast.py

```diff
@@ -0,0 +1,166 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+
+
+import dask
+import numpy as np
+import xarray as xr
+
+from climpred.bootstrap import bootstrap_hindcast
+# from climpred.constants import HINDCAST_COMPARISONS
+from climpred.prediction import compute_hindcast
+
+from . import parameterized, randn, requires_dask
+
+HINDCAST_COMPARISONS = ['m2r']  # e2r and probabilistic dont match
+METRICS = ['rmse', 'pearson_r', 'crpss']
+
+bootstrap = 8
+
+
+def _ensure_loaded(res):
+    """Compute no lazy results."""
+    if dask.is_dask_collection(res):
+        res = res.compute()
+    return res
+
+
+class Generate:
+    """
+    Generate random hind, control to be benckmarked.
```

Comment:
There's a typo in "benchmarked". I think this is also "observations" instead of "control" since we are doing hindcast here.